openhands/controller/agent_controller.py

```diff
@@ -312,6 +312,15 @@ async def _handle_message_action(self, action: MessageAction) -> None:
                 str(action),
                 extra={'msg_type': 'ACTION', 'event_source': EventSource.USER},
             )
+            # Extend max iterations when user sends a message
```

Comment:
Trivial change to trigger CI.
```suggestion
            # Extend max iterations when the user sends a message
```